data/migrations/0079_purchase_family.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 4.0.5 on 2022-06-14 15:29
+
+from django.db import migrations, models
+
+
+def migrate_product_family(apps, schema_editor):
+    Purchase = apps.get_model("data", "Purchase")
+    mapping = {
+        "VIANDES_VOLAILLES": "VIANDES_VOLAILLES",
+        "PRODUITS_DE_LA_MER": "PRODUITS_DE_LA_MER",
+        "FRUITS_ET_LEGUMES": "AUTRES",
+        "PRODUITS_CEREALIERS": "",
+        "ENTREES": "AUTRES",
+        "PRODUITS_LAITIERS": "PRODUITS_LAITIERS",
+        "BOISSONS": "BOISSONS",
+        "AIDES": "AUTRES",
+        "BEURRE_OEUF_FROMAGE": "PRODUITS_LAITIERS",
+        "PRODUITS_SUCRES": "",
+        "ALIMENTS_INFANTILES": "AUTRES",
+        "GLACES_SORBETS": "AUTRES",
+        "AUTRES": "AUTRES",
+    }
+    for purchase in Purchase.objects.all():
+        if purchase.category and mapping[purchase.category]:
```

Comment:
Cette ligne peut lever un `KeyError` pour certains cas.

Par exemple, dans la migration `0041_purchase.py` on a ajouté `PRODUITS_TRANSFORMES` comme `category` possible (qui a été enlevé par la suite). Si un achat dans notre DB a cette catégorie, au moment de la migration on aurait un `KeyError`.

```suggestion
        if purchase.category and purchase.category in mapping:
```